sdk/spring/azure-spring-boot/src/test/java/com/microsoft/azure/spring/autoconfigure/aad/AADAppRoleAuthenticationFilterTest.java

```diff
@@ -105,8 +105,8 @@ public void testDoFilterGoodCase()
         assertNull("Authentication has not been cleaned up!", SecurityContextHolder.getContext().getAuthentication());
     }
 
-    @Test(expected = ServletException.class)
-    public void testDoFilterShouldRethrowJWTException()
+    @Test
+    public void testDoFilterShouldNotRethrowJWTException()
```

Comment:
Do we need to add some verification logic here?

Reply:
Updated.